src/share/poudriere/common.sh

```diff
@@ -3844,6 +3849,8 @@ prepare_ports() {
 		# Link this build as the /latest
 		ln -sfh ${BUILDNAME} ${log%/*}/latest
 
+		DEBUG_LOG=${log}/logs/debug.log
```

Comment:
I think this should not be defined by default or only if WITH_DEBUG_LOGGING set or something. The msg_debug can be quite verbose and may impact performance by writing to this file.

I need to run this locally for a while to get a better idea on it though.